### PR TITLE
Start version 1.3.1

### PR DESCRIPTION
**Pull request by myungjoo**
As we had dinner gathering last week,
I'm moving the version forward.

Added versioning script for future use.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Summary:	gstreamer plugins for neural networks
 # 2. Tizen  : ./packaging/nnstreamer.spec
 # 3. Android: ./jni/nnstreamer.mk
 # 4. Meson  : ./meson.build
-Version:	1.3.0
+Version:	1.3.1
```
Comment (dongju-chae) [inline packaging/nnstreamer.spec]: Missing updates in line 580?

Comment (myungjoo) [reply packaging/nnstreamer.spec]: Script updated to add changelog.

File: tools/development/reversion.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	echo "Parameter not given property"
+	echo "$0 old-major old-mid old-minor new-version(full) \"Name <Email>\""
+	exit -1
+}
+
+if [ -z "$1" ]; then paramerr; fi
+if [ -z "$2" ]; then paramerr; fi
+if [ -z "$3" ]; then paramerr; fi
+if [ -z "$4" ]; then paramerr; fi
+if [ -z "$5" ]; then paramerr; fi
```
Comment (kparichay) [inline tools/development/reversion.sh]: `if [ "$#" -ne 5 ]; then paramerr; fi` ?

Comment (myungjoo) [reply tools/development/reversion.sh]: Fixed.

**Pull request merged**